README.md

```diff
@@ -1,3 +1,16 @@
-# react-lottie
+# @crello/react-lottie
 
-Example [https://crello.github.io/react-lottie/](https://crello.github.io/react-lottie/)
\ No newline at end of file
+Example [https://crello.github.io/react-lottie/](https://crello.github.io/react-lottie/)
+
+# package build scripts
```

Comment:
I think we should leave here only how to build it. Because only we can publish it =)